### PR TITLE
Add permission to enable verbose on join

### PR DESCRIPTION
**Pull request by Axionize**

### CODE REVIEW

**Comment by ManInMyVan** [comment]
Add it to the plugin yml

**Comment by Axionize** [comment]
Verbose wasn't in the plugin.yml so I chose not to add this one either. Are you certain I should add verbose on join permission to the plugin.yml when the verbose pemrission itself isn't?

**Comment by ManInMyVan** [comment]
> Verbose wasn't in the plugin.yml so I chose not to add this one either. Are you certain I should add verbose on join permission to the plugin.yml when the verbose pemrission itself isn't?

add both

**Comment by Axionize** [comment]
What about the gradle buildscript? Alerts and alert on join is likewise in it but verbose is not?

**Comment by ManInMyVan** [comment]
> What about the gradle buildscript? Alerts and alert on join is likewise in it but verbose is not?

add verbose then?

**Comment by Axionize** [comment]
Supersedes and includes https://github.com/GrimAnticheat/Grim/pull/1840

File: src/main/java/ac/grim/grimac/events/packets/PacketPlayerJoinQuit.java [hunk 1]
```diff
@@ -36,6 +36,9 @@ public void onUserLogin(UserLoginEvent event) {
         }
         if (player.hasPermission("grim.alerts") && player.hasPermission("grim.alerts.enable-on-join")) {
             GrimAPI.INSTANCE.getAlertManager().toggleAlerts(player);
+            if (player.hasPermission("grim.verbose") && player.hasPermission("grim.verbose.enable-on-join")) {
+                GrimAPI.INSTANCE.getAlertManager().toggleVerbose(player);
+            }
```
Comment (ManInMyVan) [inline src/main/java/ac/grim/grimac/events/packets/PacketPlayerJoinQuit.java:41]: this shouldn't be inside alerts, alerts perm isn't needed to toggle verbose

**Review by ManInMyVan — CHANGES_REQUESTED** [review]

**Pull request merged**